packages/jaeger-ui/src/components/TracePage/TraceSpanView/index.tsx

```diff
@@ -174,7 +174,7 @@ export default class TraceSpanView extends Component<Props, State> {
     return (
       <div>
         <h3 className="title--TraceSpanView"> Trace Tabular View</h3>
-        <Row type="flex" style={{ marginTop: '8px' }}>
```

Comment:
what did `type="flex"` do? Seems important.

Reply:
The prop was removed and it seems it's no longer needed. In v3, it was used to [control whether to set `display: flex`](https://github.com/ant-design/ant-design/blob/3.26.0/components/grid/style/index.less#L13) on the row but in v4 this seems to be [unconditional](https://github.com/ant-design/ant-design/blob/4.24.8/components/grid/style/index.less).